perf_test/blas/blas2/gemv_notes.txt

```diff
@@ -0,0 +1,36 @@
+FROM -- KokkosBlas2_gemv_perf_test.cpp 
+
+
```

Comment:
Is this file needed?

Reply:
@srajama1 - no, it is not.  It's just notes I made during the process.  I'll remove it.